apps/api/pages/api/users/_get.ts

```diff
@@ -39,6 +51,30 @@ export async function getHandler(req: NextApiRequest) {
   const where: Prisma.UserWhereInput = {};
   // If user is not ADMIN, return only his data.
   if (!isAdmin) where.id = userId;
+
+  if (req.query.email) {
+    const validationResult = schemaQuerySingleOrMultipleUserEmails.safeParse(req.query);
```

Comment:
We can simplify by using parse. `defaultResponder` already handles validations errors.
```suggestion
    const validationResult = schemaQuerySingleOrMultipleUserEmails.parse(req.query);
```

Reply:
Oh Cool! 👍